roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -0,0 +1,76 @@
+# pylint: disable=missing-docstring
+from openshift_checks import OpenShiftCheck, get_var
+from openshift_checks.mixins import NotContainerizedMixin
+
+
+class DockerImageAvailability(NotContainerizedMixin, OpenShiftCheck):
```

Comment:
I think this check could work for both containerized and non-containerized installs?